src/Console/Installer.php

```diff
@@ -181,6 +187,8 @@ public static function setSecuritySalt($dir, $io)
     {
         $newKey = hash('sha256', Security::randomBytes(64));
```

Comment:
Should I remove the injection of the salt key into app.php alltogether and only keep it for the generated app_local.php?

Reply:
What about this question. Any core team comments?

Reply:
It should be no secrets in git repo. So `app.php` should have no secrets.